integration/test.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python3
```

Comment:
can you add a high level comment about what's happening in this file?  Something like "Spin up all of the servers using each tracer and send a request that is expected to be passed between all of the servers.  Then, use the Lightstep API to query for the trace describing that request, and ensure it contains all of the servers.  If the trace is incomplete, then the context did not correctly propagate between all of the servers."

Also I'm confused about what the environment var LS_SERVICE_NAME does here - can you add a comment to that effect somewhere?

Reply:
Added a comment, the integration test itself is instrumented by the opentelemetry-launcher which uses LS_SERVICE_NAME when reporting the `service.name`